README.md

```diff
@@ -1,4 +1,5 @@
 # Next dapp starter kit
+
 This is a template meant to kickstart dapps.
 
 ![Next dapp starter kit](/public/next-dapp-starter-kit.png)
```

Comment:
Should we remove this?

Reply:
No, I think it's okay, we should that we used our dapp starter kit